### DISCUSSION TITLE
Is it supported EOR/ndata ？

### DISCUSSION THREAD

**Discussion by HparkeY** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1710] [General]
I use data channel to transfer big data, currently support eor/ndata processing?
tanks!

**Comment by disa6302** [comment]
We do not support this yet. Looking at the usrsctp documentation it seems to require just setting the `MSG_EOR` flag in the send() call. Will update this thread when we have mroe info.

**Comment by HparkeY** [comment]
Thank you for your answer.
When is it expected to be supported?
